coral/src/app/layout/main-navigation/MainNavigationLink.tsx

```diff
@@ -50,6 +50,8 @@ function MainNavigationLink(props: MainNavigationLinkProps) {
       marginLeft={"-l3"}
       marginBottom={"3"}
       paddingBottom={"3"}
+      /*console defaults to root size, which is 14px*/
```

Comment:
I think those comments are not very relevant for anyone who does not work for Aiven, maybe it's superfluous ^^

Reply:
Hm... true 🤔  
Maybe introducing a global var with a good name would be a good idea?
Otherwise it's just a random magic number appearing. 

And tbh, there's a very good change that I'll see that in 2 month from now and be like: "What? Why did I do that?" 😳 🙈

Reply:
Yeah, I'm still not 100% convinced that this change in font size is necessary tbh ^^'

Reply:
I mean... we could remove it for now, but as soon as Klaw should be part of console, we'll need it. But we can still do it then, right? 🤔

Reply:
Ah, you know what. I'll remove it and when the times comes, maybe there's a navigation link component from DS and we're aligned or we're going back and add styles then!